servers/rendering/renderer_rd/storage_rd/render_scene_data_rd.h

```diff
@@ -105,10 +105,12 @@ class RenderSceneDataRD {
 		float radiance_inverse_xform[12];
 
 		float ambient_light_color_energy[4];
+		float reflection_color[3];
 
 		float ambient_color_sky_mix;
 		uint32_t use_ambient_light;
 		uint32_t use_ambient_cubemap;
+		uint32_t use_reflection_color;
 		uint32_t use_reflection_cubemap;
```

Comment:
Change spacing to reflect 4-byte alignment as required by shaders:

```suggestion
		float ambient_light_color_energy[4];
		
		float reflection_color[3];
		float ambient_color_sky_mix;
		
		uint32_t use_ambient_light;
		uint32_t use_ambient_cubemap;
		uint32_t use_reflection_color;
		uint32_t use_reflection_cubemap;
```